hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -1389,11 +1389,14 @@ public void refresh(OmKeyInfo key) throws IOException {
     refreshPipeline(Arrays.asList(key));
   }
 
-  public static boolean isKeyDeleted(String key, Table keyTable) {
+  public static boolean isKeyDeleted(String key, Table keyTable)
```

Comment:
The purpose of method is only check in cache,  not in table. And as per caller, its checking in cache while iterate the table keys.

Reply:
> The purpose of method is only check in cache, not in table. And as per caller, its checking in cache while iterate the table keys.

I have reverted the change.